### PR TITLE
Feature/OP-1086: Default Filter for Agency Users - Search Page

### PR DESCRIPTION
**Pull request by gzhou1**
- Set default value for Agency Select field if `current_user` is an agency user.
- Includes unittests for `request.views.view_all` function route and `SearchRequestsForm` class.

### CODE REVIEW

File: tests/forms.py [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    def test_constructor(self):
+        """
+        Test class SearchRequestsForm constructor with anonymous user and logged in agency user.
+        """
+        # anonymous user
+        form = SearchRequestsForm()
+        self.assertEqual(form.agency_ein.default, None)
+
+        # agency user
+        login_user(self.agency_user)
```
Comment (Palisand) [inline tests/forms.py]: Does flask_login provide a context manager for this? If not, it is probably a good idea to make one that calls logout on exit. For now, you can also just include a `# TODO` if you'd like.

File: tests/request.py [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+    def test_view_all_agency(self, render_template_patch, search_requests_form_patch):
+        """
+        Test render_template in request.views.view_all is called once for logged in agency user.
+
+        :param render_template_patch: patch render_template method from request.views
+        :param search_requests_form_patch: patch SearchRequestsForm form object from request.views
+        """
+        # login agency_user
+        with self.client as client:
+            with client.session_transaction() as session:
```
Comment (Palisand) [inline tests/request.py]: REMINDER: `login_user_for_client_session(client)`

File: tests/request.py [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+                )
+            )
+
+    @patch('app.request.views.SearchRequestsForm')
+    @patch('app.request.views.render_template', return_value=jsonify({}))
+    def test_view_all_anon(self, render_template_patch, search_requests_form_patch):
+        """
+        Test render_template in request.views.view_all is called once for anonymous user.
+
+        :param render_template_patch: patch render_template method from request.views
```
Comment (Palisand) [inline tests/request.py]: these parameters descriptions are redundant, please remove (do the same for `test_view_all_agency`)

File: tests/forms.py (hunk 1 above)
Comment (gzhou1) [inline tests/forms.py]: flask_login does not provide a context manager. I have included a `# TODO` for now.

File: tests/request.py (hunk 3 above)
Comment (gzhou1) [inline tests/request.py]: Removed parameter descriptions.

File: tests/request.py (hunk 2 above)
Comment (gzhou1) [inline tests/request.py]: Created a `# TODO`.

**Pull request merged**